index.html

```diff
@@ -115,6 +115,38 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+                <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+                <div class="card">
+                  <p class="name">Oluyanju Oluwaseyi emperor6inch</p>
+                  <p class="contact">
+                    <i class="fab fa-twitter"></i>
+                    <a href="www.twitter.com" target="_blank">emperor6inch</a>
+                  </p>
+                  <p class="about">Learning how the world works through computers.</p>
+                  <div class="resources">
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                      <li>
+                        <a href="#" target="_blank" title="First Resource">
+                          Learn Python3 The Hard Way
+                        </a>
+                      </li>
+                      <li>
+                        <a href="#" target="_blank" title="Second Resource">
+                          Freecodecamp.com
+                        </a>
+                      </li>
+                      <li>
+                        <a href="#" target="_blank" title="Third resource">
+                          Automating the Boring stuff with Python
+                        </a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+                <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
Please change this comment as well. 
`<!-- ________*END TEMPLATE* Contributor card END ________ -->` 
to `<!-- ________ Your_Name Contributor card END ________ -->`